### PR TITLE
WD: add GitBook project to Writing Day list

### PR DESCRIPTION
**Pull request by rosewms**
Needs to be reviewed by someone from GitBook before it goes live.

----
📚 Documentation preview 📚: https://writethedocs-www--2114.org.readthedocs.build/

### CODE REVIEW

**Comment by addisonschultz** [comment]
Hi @rosewms ! Thanks so much for helping us get this project up on the site! We're excited 😄 

There's two small typos, but other than that it looks good to me. Feel free to merge!

File: docs/conf/portland/2024/writing-day.rst [hunk 1]
```diff
@@ -130,6 +130,27 @@ Coordinated by Kenzie Woodbridge, they/them. Let's help each other get ready for
...
+
+That's not all, GitBook is bringing three Writing Day challenges. No matter your experience level, these challenges are designed to guide you through the tasks necessary to publish your first, but hopefully not last, GitBook knowledge site! The content on your knowledge site is up to you, it can be documentation, a company knowledge base, or something to help you plan the rest of your sprint. GitBook staff will be around to answer any questions you have.
+
+Writing Day challenges:
+
+- Publish your first site with GitBook
+- Install the Slack Integration and capture your first snippet
+- Import your docs from an existing repo or an open source project
+
+It gets better! For every challenge completed, you'll receive one (1) entry for our raffle to win an epic swag box! That mean if you complete all 3 challeges, you'll get three (3) entries. More information about the raffle entry is available during each session.
```
Comment (addisonschultz) [inline docs/conf/portland/2024/writing-day.rst]: `challeges` > `challenges`

Comment (addisonschultz) [inline docs/conf/portland/2024/writing-day.rst]: `mean` > `means`

**Review by addisonschultz — APPROVED** [review]
Looks good to me!

**Review by plaindocs — APPROVED** [review]

**Pull request merged**